doc/tutorials/specification.md

```diff
@@ -503,6 +516,30 @@ database/dialect =
  meta:/check/enum = #4
 ```
 
+## Cleanup after the tutorial
```

Comment:
Hey @flo91 , as I can see you put more effort into T1, so I think we should merge yours ( #4315 ) first, and I'll take the time to rebase and resolve the merge conflicts. Would that be ok with you?
Maybe I can do the shellrecorder stuff to balance work out and so we don't loose any more time with merging #4315 .

I hope this course of action is also OK for you, @kodebach .

Reply:
Hello Lukas,
as discussed today, I finished my PR #4315 and added the shellrecorder test.
You probably have to adapt it so that your changes are also covered by the test.
My PR is already rebased with current master.

Reply:
@kodebach can we merge #4315 now? I would then rebase this PR.

Reply:
The build server did not pass yet and @flo91 needs to mark the PR as "ready to merge".

Reply:
I posted some notes in #4315 on how the build problem can be fixed.